appyters/KEA3_Appyter/KEA3-Appyter.ipynb

```diff
@@ -169,95 +169,110 @@
     "    all_prePPI = []\n",
     "    all_PTMsigDB = []\n",
     "    \n",
-    "    for kinase in range(0, 10):\n",
+    "    # Length of all the libraries (for calculation of )\n",
+    "    max_ranks = {}\n",
+    "    for library in range(2,13):\n",
+    "        lib_name = (list(results.keys())[library])\n",
+    "        max_ranks.update({lib_name: len(list(results.values())[library])})\n",
```

Comment:
Same thing here, it would just be: `max_ranks[libname] = len(results[lib_name])`